SPECS/busybox/busybox.spec

```diff
@@ -94,6 +94,9 @@ install -m 644 docs/busybox.petitboot.1 %{buildroot}/%{_mandir}/man1/busybox.pet
 %{_mandir}/man1/busybox.petitboot.1.gz
 
 %changelog
+* Wed Jul 05 2023 Andrew Phelps <anphel@microsoft.com> - 1.35.0-5
+- Bump release to rebuild against glibc 2.35-4
+
 * Fri Oct 07 2022 Andy Caldwell <andycaldwell@microsoft.com> - 1.35.0-4
```

Comment:
Note: it looks like the release was not actually bumped to 4 with this previous change. I've skipped release 4 here and gone directly to 5.

Reply:
Oh, my bad, sorry, must have missed it.  It's not worth cutting a `-4` jumping to `-5` is fine.